pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
     "ruamel.yaml>=0.16",
     "scipy>=1.4",
     "importlib-resources; python_version < '3.9'",  # TODO: remove when minimum python version is 3.9
+    "threadpoolct"
```

Comment:
Do you like to have lower bound versions pinned here?

Reply:
@rly thoughts?

Reply:
Moved to https://github.com/hdmf-dev/hdmf-zarr/pull/111